### DISCUSSION TITLE
import en_ner_bionlp13cg_md results in error --> AttributeError: type object 'Language' has no attribute 'component'

### DISCUSSION THREAD

**Discussion by Rd0394** [explosion/spaCy#7390] [Help: Installation] [answered]
Code:
!pip install spacy
!pip install scispacy
!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.0/en_ner_bionlp13cg_md-0.2.0.tar.gz                #install an NER model trained on bionlp13c corpus

#Environment used:

* Operating System: Ubuntu 20.04.2 LTS
* Python Version Used: Python 3.8.5
* spaCy Version Used: Spacy 3.0.4

![git](https://user-images.githubusercontent.com/78722507/110588559-30964300-819b-11eb-8c0e-86df99e4bfbe.png)

I need to load the bionlp model and filter out the biological terms in my dataset

**Comment by adrianeboyd** [comment, marked as answer]
Hi, you need to update your model URL to the one for the version of scispacy that supports spacy v3. See:

https://github.com/allenai/scispacy/#available-models

The older models for spacy v2 / scispacy 0.2.x/0.3.x are not compatible with the spacy v3.

**Reply by Rd0394** [reply]
It worked!!! thanks. I could load the model and extract desired output. Yet i received this warning message. Is there anything to be considered in this warning?
![Screenshot from 2021-03-10 17-01-16](https://user-images.githubusercontent.com/78722507/110623095-5e42b280-81c2-11eb-90ff-25f10355755b.png)

**Reply by adrianeboyd** [reply]
You want to update all the scispacy models from the table in that link. That model is not going to load with spacy v3.

**Reply by Rd0394** [reply]
Thanks !! It worked without any warnings!!
